css/hotel.scss

```diff
@@ -37,6 +37,13 @@
   justify-content: center;
 }
 
+#hotel-img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  max-height: 70vh;
```

Comment:
This makes things real easy.  The image will take whatever height it can, keeping scaling, up to either 100% width of the parent element, or 70% of the browser height.